gradle/dependencies.gradle

```diff
@@ -15,367 +15,386 @@
+                    compile(libraries.slf4j_api) { version { strictly versions.slf4j } }
+                    compile(libraries.jcl_to_slf4j) { version { strictly versions.slf4j } }
+                    compile(libraries.jul_to_slf4j) { version { strictly versions.slf4j } }
+                    compile(libraries.log4j_to_slf4j) { version { strictly versions.slf4j } }
 
-libraries.ivy = 'org.apache.ivy:ivy:2.2.0'
+                    // lock transitive dependencies that are part of the Gradle distribution
+                    compile('org.jetbrains:annotations') { version { strictly '13.0' } }
+                    compile('org.apache.ant:ant-launcher') { version { strictly versions.ant } }
+                    compile('org.ow2.asm:asm-util') { version { strictly versions.asm } }
+                    compile('com.esotericsoftware.minlog:minlog') { version { strictly '1.2' } }
+                    compile('org.sonatype.aether:aether-api') { version { strictly versions.aether } }
+                    compile('org.sonatype.aether:aether-impl') { version { strictly versions.aether } }
+                    compile('org.sonatype.aether:aether-spi') { version { strictly versions.aether } }
+                    compile('org.sonatype.aether:aether-util') { version { strictly versions.aether } }
+                    compile('com.google.api-client:google-api-client') { version { strictly '1.23.0' } }
+                    compile('com.google.http-client:google-http-client') { version { strictly '1.23.0' } }
+                    compile('com.google.http-client:google-http-client-jackson2') { version { strictly '1.23.0' } }
+                    compile('com.google.oauth-client:google-oauth-client') { version { strictly '1.23.0' } }
+                    compile('org.hamcrest:hamcrest-core') { version { strictly '1.3' } }
+                    compile('org.apache.httpcomponents:httpcore') { version { strictly '4.4.4' } }
+                    compile('com.beust:jcommander') { version { strictly '1.47' } }
+                    compile('org.apache.maven:maven-aether-provider') { version { strictly versions.maven } }
+                    compile('org.apache.maven:maven-artifact') { version { strictly versions.maven } }
+                    compile('org.apache.maven:maven-compat') { version { strictly versions.maven } }
+                    compile('org.apache.maven:maven-model') { version { strictly versions.maven } }
+                    compile('org.apache.maven:maven-model-builder') { version { strictly versions.maven } }
+                    compile('org.apache.maven:maven-plugin-api') { version { strictly versions.maven } }
+                    compile('org.apache.maven:maven-repository-metadata') { version { strictly versions.maven } }
+                    compile('org.apache.maven:maven-settings') { version { strictly versions.maven } }
+                    compile('org.apache.maven:maven-settings-builder') { version { strictly versions.maven } }
```

Comment:
Maven is another 'platform' that we should define somehow, to avoid these sorts of lists. I guess it gives a good example of how a constraint that matches only `group` would be useful.